LayoutTests/http/tests/workers/service/shownotification-invalid-data.html

```diff
@@ -0,0 +1,58 @@
+<head>
+<script src="/resources/js-test-pre.js"></script>
+<script src="/resources/notifications-test-pre.js"></script>
+<script>
+
+function failTheTest(msg)
+{
+    testFailed(msg);
+    if (testRunner)
+        testRunner.notifyDone();
+}
+
+if (window.testRunner) {
+    testRunner.waitUntilDone();
```

Comment:
with js-test.js, you should be using:
`jsTestIsAsync = true;`

not `testRunner.waitUntilDone();`